courses/wagtail_hooks.py

```diff
@@ -0,0 +1,14 @@
+"""Wagtail Snippets for courses app"""
+from wagtail.snippets.models import register_snippet
+from wagtail.snippets.views.snippets import DeleteView, SnippetViewSet
+
+from courses.models import CourseTopic
+
+
+class CourseTopicSnippet(SnippetViewSet):
+    """Snippet for CourseTopic"""
+
+    model = CourseTopic
+
+
+register_snippet(CourseTopicSnippet)
```

Comment:
Yes, Moved from modeladmins to snippets. modeladmins are moved out of wagtail and wagtail recommends using snippets. I just wanted to avoid an extra dependency here.

Reply:
I think the main reason initially was to keep the topics on the same levels as Snippets, But now the topics are moved under Snippets as a child.

See the previous look:
<img width="202" alt="image" src="https://github.com/mitodl/mitxpro/assets/34372316/2e9b1a44-3247-4806-b556-0ba77762a58c">

Reply:
Updated in https://github.com/mitodl/mitxpro/pull/2830/commits/6614b627ad2856d4a6f0511074b5865606e9a1bf